ratbagd/ratbagd-button.c

```diff
@@ -149,6 +149,60 @@ static int ratbagd_button_set_special(sd_bus *bus,
 	return 0;
 }
 
+static int ratbagd_button_get_key(sd_bus *bus,
+				      const char *path,
+				      const char *interface,
+				      const char *property,
+				      sd_bus_message *reply,
+				      void *userdata,
+				      sd_bus_error *error)
+{
+	struct ratbagd_button *button = userdata;
+	unsigned int key;
+
+	unsigned int *modifiers = 0;
+	size_t *sz = 0;
+        key = ratbag_button_get_key(button->lib_button, modifiers, sz);
+
+	verify_unsigned_int(key);
+
+	CHECK_CALL(sd_bus_message_append(reply, "(uv)",
+					 RATBAG_BUTTON_ACTION_TYPE_KEY,
+					 "u",
+					 key));
+
+	return 0;
+}
+
+static int ratbagd_button_set_key(sd_bus *bus,
+				      const char *path,
+				      const char *interface,
+				      const char *property,
+				      sd_bus_message *m,
+				      void *userdata,
+				      sd_bus_error *error)
+{
+	struct ratbagd_button *button = userdata;
+	unsigned int key;
+	int r;
```

Comment:
Nit-pick
```suggestion
	int rc;
```

Reply:
Are you sure? The entire file uses "r" as return variable. Maybe it's an old code style, I don't know.

Reply:
Oh, the whole `ratbagd` directory has `r` and not `rc`. Sorry.